### ISSUE TITLE
How to send transactions for tokens ? 

### ISSUE THREAD

**Issue by RocketBarage** [simolus3/web3dart#66]
I have a contract address, how to send (example: 0xfaa27f641d559a10c0e79145342ce1f9c799ca70) a transaction

**Comment by simolus3** [comment]
I think I need more details. Are you talking about a standard ERC20 token?

**Comment by RocketBarage** [comment]
> I think I need more details. Are you talking about a standard ERC20 token?

Yes

**Comment by simolus3** [comment]
Right. For those you need the abi (a simple json file) of the contract, like available [here](https://gist.github.com/veox/8800debbf56e24718f9f483e1e40c35c). You can then create a contract via
```dart
final client = Web3Client(...);
final contract =
      DeployedContract(ContractAbi.fromJson(<abi-code-as-string>, 'name'), <contract-address>);
``` 

And you can call methods on the contract. Sending tokens could look like
```dart
final transfer = contract.function('transfer');
await client.sendTransaction(
  credentials,
  Transaction.callContract(
    contract: contract,
    function: transfer,
    parameters: [EthereumAddress('receiving address'), BigInt.from(tokensToSend)],
  ),
);
```

I know that this is a lot of work at the moment, and there should be apis to make interacting with a contract easier. I started some work to generate Dart apis from contracts over [here](https://github.com/ethereumdart/contracts), but I didn't have time to publish that yet.
